### PR TITLE
Update handlebars definition files for handlebars 3.0.3

### PR DESCRIPTION
**Pull request by CalvinFernandez**

### CODE REVIEW

**Comment by vvakame** [comment]
*ember/ember.d.ts*

to author(@jedmao). could you review this PR?
:+1: or :-1:?

check list

* [ ] pass the Travic-CI test?

---

*handlebars/handlebars-2.0.0.d.ts*

check list

* [ ] is collect [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)?
  * https://www.npmjs.com/package/handlebars
  * http://bower.io/search/?q=handlebars
  * others?
* [ ] has a [test file](http://definitelytyped.org/guides/contributing.html#tests)? (handlebars/handlebars-2.0.0-tests.ts or others)
* [ ] pass the Travic-CI test?

---

*handlebars/handlebars.d.ts*

to author(@borisyankov). could you review this PR?
:+1: or :-1:?

check list

* [ ] pass the Travic-CI test?

File: handlebars/handlebars-2.0.0.d.ts [hunk 1]
```diff
@@ -0,0 +1,184 @@
+// Type definitions for Handlebars 1.0
```
Comment (jedmao) [inline handlebars/handlebars-2.0.0.d.ts]: Should this comment not say Handlebars 2.0?

File: handlebars/handlebars.d.ts [hunk 2]
```diff
@@ -1,12 +1,66 @@
-// Type definitions for Handlebars 1.0
+// Type definitions for Handlebars v3.0.3
```
Comment (jedmao) [inline handlebars/handlebars.d.ts]: I believe we should be preserving version 1.0 definitions as handlebars-1.0.d.ts. Then, you can keep this handlebars.d.ts as it is here.

Comment (CalvinFernandez) [inline handlebars/handlebars.d.ts]: Okay I've moved handlebars-2.0.0 to handlebars-1.0.0

File: handlebars/handlebars-2.0.0.d.ts (hunk 1 above)
Comment (CalvinFernandez) [inline handlebars/handlebars-2.0.0.d.ts]: Per your comment below, I've changed the file name to handlebars-1.0.0

File: handlebars/handlebars-1.0.0.d.ts [hunk 3]
```diff
@@ -0,0 +1,184 @@
+// Type definitions for Handlebars 1.0
```
Comment (jedmao) [inline handlebars/handlebars-1.0.0.d.ts]: I think you might have to do 2 separate commits to preserve history here. The first commit would be a `git mv` command to rename this file from `handlebars.d.ts` to `handlebars-1.0.0.d.ts`. The 2nd commit would be introducing a new `handlebars.d.ts` with the newest definitions.

Comment (CalvinFernandez) [inline handlebars/handlebars-1.0.0.d.ts]: done

**Comment by jedmao** [comment]
:+1:

**Comment by vvakame** [comment]
@jedmao thanks!
@CalvinFernandez thanks mate!
